vault/token_store_test.go

```diff
@@ -3588,11 +3669,75 @@ func TestTokenStore_RoleAllowedPolicies(t *testing.T) {
 		t.Fatalf("bad: %#v", resp)
 	}
 
+	// test glob matching with allowed_policies_glob
+	req = logical.TestRequest(t, logical.UpdateOperation, "roles/test")
+	req.ClientToken = root
+	req.Data = map[string]interface{}{
+		"allowed_policies":      "",
```

Comment:
If it's not the existing value for `allowed_policies` from the previous test remains in the role. Technically the test passes either way (and would fail without the glob being added), but it's more explicit that it's not replying on the value from the previous test this way.

Reply:
Ah I missed that, yeah explicit resetting then makes sense 👍